README.md

```diff
@@ -1,11 +1,16 @@
 [Sangria](http://sangria-graphql.org/) [circe](http://circe.io) marshalling.
 
-![Continuous Integration](https://github.com/sangria-graphql/sangria-circe/workflows/Continuous%20Integration/badge.svg) [![Build Status](https://travis-ci.org/sangria-graphql/sangria-circe.svg?branch=master)](https://travis-ci.org/sangria-graphql/sangria-circe) [![Coverage Status](http://coveralls.io/repos/sangria-graphql/sangria-circe/badge.svg?branch=master&service=github)](http://coveralls.io/github/sangria-graphql/sangria-circe?branch=master) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.sangria-graphql/sangria-circe_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.sangria-graphql/sangria-circe_2.11) [![License](http://img.shields.io/:license-Apache%202-brightgreen.svg)](http://www.apache.org/licenses/LICENSE-2.0.txt) [![Join the chat at https://gitter.im/sangria-graphql/sangria](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/sangria-graphql/sangria?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+![Continuous Integration](https://github.com/sangria-graphql/sangria-circe/workflows/Continuous%20Integration/badge.svg)
+[![Build Status](https://travis-ci.org/sangria-graphql/sangria-circe.svg?branch=master)](https://travis-ci.org/sangria-graphql/sangria-circe)
+[![Coverage Status](http://coveralls.io/repos/sangria-graphql/sangria-circe/badge.svg?branch=master&service=github)](http://coveralls.io/github/sangria-graphql/sangria-circe?branch=master)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.sangria-graphql/sangria-circe_2.12/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.sangria-graphql/sangria-circe_2.12)
+[![License](http://img.shields.io/:license-Apache%202-brightgreen.svg)](http://www.apache.org/licenses/LICENSE-2.0.txt)
+[![Join the chat at https://gitter.im/sangria-graphql/sangria](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/sangria-graphql/sangria?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 SBT Configuration:
 
 ```scala
-libraryDependencies += "org.sangria-graphql" %% "sangria-circe" % "1.2.1"
+libraryDependencies += "org.sangria-graphql" %% "sangria-circe" % "1.3.1"
```

Comment:
Maybe we could use the following to avoid having to update the README at each update?

```suggestion
libraryDependencies += "org.sangria-graphql" %% "sangria-circe" % "<last version>"
```